frontend/www/js/omegaup/arena/submissions.ts

```diff
@@ -59,6 +75,149 @@ export function submitRunFailed({
   }
 }
 
+export function showSubmission({
+  request,
+  target,
```

Comment:
```suggestion
  request: { request, target },
```

así se puede des-estructurar un miembro anidado.

Reply:
Listo